docs/did-spec/spec.md

```diff
@@ -149,6 +150,19 @@ Hence, a light DID can be migrated to a full DID and the KILT blockchain ensures
 
 Once upgraded, a light DID becomes unusable, meaning that all the operations involving authentication of the DID subject must be performed using the currently active authentication key of the full DID.
 
+### Add a web3 name to a full DID
+
+Full DIDs can optionally claim a unique user-friendly alias called **Web3 name**, similar to what domain names are for IP addresses.
+Each DID can only claim at most one web3 name, and each web3 name can be only claimed by one DID, with the KILT blockchain ensuring the unicity property.
+
+A Web3 name can only contain lowercase ASCII characters, digits, and the symbols `-` and `_`.
+The choice of restricting the set of characters to a subste of the ASCII charset with the addition of digits and some symbols ensures that no two Web3 name can look alike, as it would be the case of uppercase `O` and the digit `0`.
```

Comment:
```suggestion
The choice of restricting the set of characters to a subset of the ASCII charset with the addition of digits and some symbols ensures that no two Web3 name can look alike, as it would be the case of uppercase `O` and the digit `0`.
```

Reply:
https://github.com/KILTprotocol/kilt-did-driver/pull/27/commits/31d429616035b947eb549b3383749f52c28deb03